tests/system/conftest.py

```diff
@@ -126,7 +126,7 @@ def pytest_runtest_makereport(item, call):
                 arguments[key] = str(value)
         extra.append(pytest_html.extras.json(arguments, name="Test arguments"))
 
-        if "cluster" in report.markers:
+        if "cluster" in report.markers and "one_manager_agent_env" not in report.markers:
```

Comment:
Why does `one_manager_agent_env` include the cluster marker? It seems that removing the marker would result in a more maintainable approach for this particular case.

In addition, handling the logs from cluster environment using markers is not the best approach. We should consider to change this logic in order to make it more error resistant, including a minor inventory validation

Reply:
fixed in https://github.com/wazuh/wazuh-qa/pull/4782/commits/0baa8e303905ec25290fadc055250847f3155749